pkg/go/go.mod

```diff
@@ -1,31 +1,31 @@
 module github.com/openfga/language/pkg/go
 
-go 1.20
+go 1.21.6
```

Comment:
For the language, please keep it @ 1.20 at least until [go1.22](https://tip.golang.org/doc/go1.22) is released.

Reply:
Do we not need to bump this minimum to ensure security updates are applied for the runtime? `1.21.5` includes security updates for the `net/http` and `path/filepath` packages.

Reply:
The CLI we are forced to by the server (we bundle it as a dependency), but we should aim to support at least the versions supported by Go for security updates (currently 1.20 + 1.21, soon to be 1.21 + 1.22). [Go Release Policy](https://go.dev/doc/devel/release#policy)

> Each major Go release is supported until there are two newer major releases. For example, Go 1.5 was supported until the Go 1.7 release, and Go 1.6 was supported until the Go 1.8 release. We fix critical problems, including [critical security problems](https://go.dev/security), in supported releases as needed by issuing minor revisions (for example, Go 1.6.1, Go 1.6.2, and so on).

Reply:
The fixes in go 1.21.5 were included in [1.20.12](https://github.com/golang/go/issues?q=milestone%3AGo1.20.12+label%3ACherryPickApproved)